src/main/java/tc/oc/pgm/listeners/PGMListener.java

```diff
@@ -303,5 +305,27 @@ public void nerfFishing(PlayerFishEvent event) {
         caught.setItemStack(new ItemStack(Material.RAW_FISH));
       }
     }
+
+
+  }
+
+  @EventHandler(priority = EventPriority.MONITOR)
+  public void dropItemsOnQuit(PlayerParticipationStopEvent event) {
+    MatchPlayer quitter = event.getPlayer();
+    Player player = Bukkit.getPlayer(quitter.getId());
+
+      for (ItemStack i : quitter.getInventory().getContents()) { //Drop inventory
```

Comment:
Can we not call variables with single letters? Can we call this `item` instead?

Reply:
Good idea! Will change boths "i"s to "item"